metecho/api/models.py

```diff
@@ -81,6 +81,26 @@ class TaskReviewStatus(models.TextChoices):
     CHANGES_REQUESTED = "Changes requested"
 
 
+class TaskActivityType(models.TextChoices):
+    MERGED = "Merged"
+    APPROVED = "Approved"
+    CLOSED = "Closed"
+    TEST_ORG_CREATED = "Test Org created"
+    TEST_ORG_DELETED = "Test Org deleted"
+    DEV_ORG_CREATED = "Dev Org created"
+    DEV_ORG_DELETED = "Dev Org deleted"
+    SUBMITTED_FOR_TESTING = "Submitted for testing"
+    CHANGES_RETRIEVED = "Changes retrieved"
+    CHANGES_REQUESTED = "Changes requested"
+    TESTER_ACCEPTED = "Tester accepted"
+    TESTER_ASSIGNED = "Tester assigned"
+    TESTER_UNASSIGNED = "Tester unassigned"
+    DEV_ACCEPTED = "Developer accepted"
```

Comment:
`TESTER_ACCEPTED` and `DEV_ACCEPTED` are not used yet but they are in place for the future invitation system